scripts/scylla-get.sh

```diff
@@ -0,0 +1,193 @@
+#!/bin/bash
+#
+# A platform agnostic Scylla server installation script.
+#
+# To install Scylla, run the following command:
+#
+# curl --proto '=https' --tlsv1.2 -sSf https://raw.githubusercontent.com/scylladb/scylla/master/scripts/scylla-get.sh | sudo bash
```

Comment:
Does `curl https:/...` not work?

Reply:
And maybe the URL can be simplified. Although of course the commit can't tell where it will end up.

Reply:
The "-sS" part is to make `curl` silent, but print out an error if it fails. The `-f` option makes `curl` fail on HTTP errors (rather than attempting to run a HTML page). The `--proto` and `--tlsv1.2` is just paranoia to enforce TLS (shamelessly copied from https://rustup.rs/).

The URL should indeed be `get.scylladb.com` or something, but I can't fix that in this commit.